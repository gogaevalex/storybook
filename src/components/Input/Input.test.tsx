import { render } from '@testing-library/react';
import { TestProvider } from '../../test/TestProvider';
import { Input, InputProps } from '.';

describe('Input', () => {
  const defaultProps: InputProps = {
    value: '',
    onChange: () => {},
  };
  it('Default render', () => {
    const { container } = render(
      <TestProvider>
        <Input {...defaultProps} />
      </TestProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
