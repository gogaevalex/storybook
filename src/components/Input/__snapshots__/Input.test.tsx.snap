// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Default render 1`] = `
<div>
  <div
    class="css-1ffo92a"
  >
    <div
      class="css-17owc11"
    >
      <input
        autocomplete="off"
        class="css-1xmsu08"
        name="input"
        placeholder=""
        type="text"
        value=""
      />
    </div>
  </div>
</div>
`;
